### PR TITLE
[Doc Improvement][Update on auth parameter][3345899] 

### PR DESCRIPTION
**Pull request by v-ganr**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e3dbeec0039b3e5ae9f9bc594f43556738d0a940/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Fauth-oauth-provider.md) | 90 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c91c20d1-477a-4aaf-9850-4b38229ee499) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ The following image provides the flow to add authentication to external browsers
     });
     ```
 
-1. The Teams clients open the URL in an external browser after replacing the placeholders for `oauthRedirectMethod` and `authId` with suitable values.
+1. The Teams clients open the URL in an external browser after replacing the placeholders automatically for `oauthRedirectMethod` and `authId` with suitable values.
```
Comment (v-deabira) [inline msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md:65]:
```suggestion
The Teams clients open the URL in an external browser after automatically replacing the placeholders for `oauthRedirectMethod` and `authId` with suitable values.
```

**Review by v-deabira — CHANGES_REQUESTED** [review]
@v-ganr, please make the change.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0d2627bd707c8bc80f19c8ba962b794ed7fe0f40/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Fauth-oauth-provider.md) | 90 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3a948a1f-d6c9-4b15-8e22-98b7b94a735e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6f459d0bb1426ee23a6f0e8c51e14f8efad4cccd/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Fauth-oauth-provider.md) | 90 | 6 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3bbae53d-408c-4bd8-a41c-3ac753fdf965) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Prasad-MSFT — APPROVED** [review]
Approved

**Review by v-deabira — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**